src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -1,82 +1,31 @@
-//            .withEmail("lydia@example.com").withAddress("little tokyo").build();
-//    public static final Person GEORGE = new PersonBuilder().withName("George Best").withPhone("9482442")
-//            .withEmail("anna@example.com").withAddress("4th street").build();
-//
-//    // Manually added
-//    public static final Person HOON = new PersonBuilder().withName("Hoon Meier").withPhone("8482424")
-//            .withEmail("stefan@example.com").withAddress("little india").build();
-//    public static final Person IDA = new PersonBuilder().withName("Ida Mueller").withPhone("8482131")
-//            .withEmail("hans@example.com").withAddress("chicago ave").build();
-//
-//    // Manually added - Person's details found in {@code CommandTestUtil}
-//    public static final Person AMY = new PersonBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY)
-//            .withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY).withTags(VALID_TAG_FRIEND).build();
-//    public static final Applicant AMY_APPLICANT =
-//            new ApplicantBuilder().withName(VALID_NAME_AMY).withPhone(VALID_PHONE_AMY).build();
-//    public static final Person BOB = new PersonBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB)
-//            .withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB).withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND)
-//            .build();
-//    public static final Applicant BOB_APPLICANT =
-//            new ApplicantBuilder().withName(VALID_NAME_BOB).withPhone(VALID_PHONE_BOB).build();
-//
-//    public static final String KEYWORD_MATCHING_MEIER = "Meier"; // A keyword that matches MEIER
-//
-//    private TypicalPersons() {} // prevents instantiation
-//
-//    /**
-//     * Returns an {@code AddressBook} with all the typical persons.
-//     */
-//    public static AddressBook getTypicalAddressBook() {
-//        AddressBook ab = new AddressBook();
-//        for (Person person : getTypicalPersons()) {
-//            ab.addPerson(person);
-//        }
-//        return ab;
-//    }
-//
-//    public static List<Person> getTypicalPersons() {
-//        return new ArrayList<>(Arrays.asList(ALICE, BENSON, CARL, DANIEL, ELLE, FIONA, GEORGE));
-//    }
-//}
+package seedu.address.testutil;
+
+import static seedu.address.testutil.TypicalApplicants.getTypicalApplicants;
+import static seedu.address.testutil.TypicalMembers.getTypicalMembers;
+
+import seedu.address.model.AddressBook;
+import seedu.address.model.person.Applicant;
+import seedu.address.model.person.Member;
+
+/**
+ * A utility class containing a list of {@code Person} objects to be used in tests.
+ */
+public class TypicalPersons {
+
+    private TypicalPersons() {
+    } // prevents instantiation
+
+    /**
+     * Returns an {@code AddressBook} with all the typical members.
+     */
+    public static AddressBook getTypicalAddressBookWithMembersApplicants() {
```

Comment:
You may want to consider creating another method in AddressBook to combine AddressBooks so that we don't need to call this method here